tests/Xamarin.Android.Net-Tests/ServerCertificateCustomValidationCallbackTest.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+using Xamarin.Android.Net;
+
+using NUnit.Framework;
+
+namespace Xamarin.Android.Net.Tests
+{
+	[Category ("InetAccess")]
+	public abstract class ServerCertificateCustomValidationCallbackTest
+	{
+		protected abstract HttpMessageHandler CreateHandlerWithCallback(Func<HttpRequestMessage, X509Certificate2?, X509Chain?, SslPolicyErrors, bool> callback);
```

Comment:
@simonrozsival so I think the only change needed, is to move these tests to:

https://github.com/xamarin/xamarin-android/tree/main/tests/Mono.Android-Tests/Xamarin.Android.Net

Then they will run on CI, and we won't need a new project. Thanks!